data-model/SDP-data-model-templates/place_sdp-template.json

```diff
@@ -1,9 +1,19 @@
 {
-  "id": "ARK/URI(?)",
-  "label": "",
+  "id": "URI",
   "type": "CV",
   "place_type": "CV",
   "uniform_name": "",
+  "alt_name": [
```

Comment:
Unlike person and work I don't think we'd need an ns_name field, but we would want the alt_name to have language mappings.

Alt names would be in general useful for places so we can have "Edessa" alongside "Urhay" "Urfa", etc. -- before even getting into the non-Latin-script versions of these names.